src/Component/Quote.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './CSS/Quote.css';
+
+function Quote() {
+  return (
+    <div className="quote">
+      <p>
+        Mathematics is not about numbers, equations, computations, or
+        <br />
+        algorithms: it is about understanding.
+      </p>
+      <p className="name">- William Paul Thurston</p>
+    </div>
+  );
+}
+
+export default Quote;
```

Comment:
@medaminedev66,

Good afternoon.

From my end here the user can navigate between pages as the navigation links for all three routes are displayed on each page. This already is implemented in the App.js component. I do not know what to do again. Can you explain better?  Thank you.

Reply:
Hi @ecekpo. The user cannot navigate between pages in your app because there are no navigation links for the routes on your pages. In your App.js component, there is only the Calculator component:

<img width="299" alt="image" src="https://user-images.githubusercontent.com/77190203/190156506-ebfd96f9-0c7e-402c-8ed7-065eab18c3fc.png">